Source/Base/ASLog.m

```diff
@@ -0,0 +1,79 @@
+//
+//  ASLog.m
+//  Texture
+//
+//  Copyright (c) 2017-present, Pinterest, Inc.  All rights reserved.
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+
+#import <AsyncDisplayKit/ASLog.h>
+
+os_log_t ASNodeLog() {
+  static os_log_t log;
```

Comment:
Doesn't this need to be initialized to some blank / NULL value in the case that the compile time gating is turned off?